requirements.txt

```diff
@@ -1,5 +1,5 @@
 async_lru
-galaxy_importer
+galaxy_importer>=3.0
```

Comment:
```suggestion
galaxy_importer>=0.3
```

Reply:
I went ahead and set it to the full version with the `file_url` parameter: `galaxy-importer==0.3.1`